README.md

```diff
@@ -803,6 +807,8 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :books: [Deep Learning - University of buffalo](https://cedar.buffalo.edu/~srihari/CSE676/index.html)
 - :books: [Machine Learning Introduction with Python](https://www.dataquest.io/path/machine-learning-intro-with-python/)
 - :speaker: [Machine Learning Guide Podcast](https://ocdevel.com/mlg)
+- :books: [An Introduction to Statistical Learning with Applications in R](http://faculty.marshall.usc.edu/gareth-james/)
```

Comment:
This link seems to be linking to other websites and it doesn't contain anything specific or helpful. Either put the direct link to the resource or remove it.

Reply:
I have changed the link to a more preferred landing page for accessing this resource. There is a "Download the second edition" button at the top allowing the end user to access it. A must-read for anybody interested in Machine Learning. Thanks.